charts/k8ssandra/Chart.yaml

```diff
@@ -3,7 +3,7 @@ name: k8ssandra
 description: |
   Provisions and configures an instance of the entire K8ssandra stack. This includes Apache Cassandra, Stargate, Reaper, Medusa, Prometheus, and Grafana.
 type: application
-version: 0.48.0
+version: 0.49.0
```

Comment:
No need to update the chart version for just the docs changes, the two aren't tied together.